scripts/benchmark/checks/simple_value_transfers.py

```diff
@@ -94,15 +83,14 @@ def execute(self) -> DefaultStat:
 
         return total_stat
 
-    def mine_blocks(self, chain: MiningChain, num_blocks: int) -> Tuple[int, int]:
+    def mine_blocks(self, chain: MiningChain, num_blocks: int, num_tx: int) -> Tuple[int, int]:
         total_gas_used = 0
         total_num_tx = 0
 
         for i in range(1, num_blocks + 1):
-            num_tx = chain.get_block().header.gas_limit // SIMPLE_VALUE_TRANSFER_GAS_COST
```

Comment:
Hm, I see where you're going, but this hard-coding approach for the number of transactions seems problematic. It requires the configuration to be precisely aware of the gas block limit in order to fill it up. Maybe the configuration parameter could be the percentage of the block to fill up? (defaulting to 1.0)

Reply:
@carver, do we really need to fill it up completely? Why don't we care about same in erc20_interact and deploy_dos?

Reply:
Yup, totally fair point. This is a fundamental change to how the benchmark works, but it's a change *toward* consistency with the other benchmarks. :+1: no change needed.